baus/subsidies.py

```diff
@@ -1128,3 +1161,76 @@ def subsidized_residential_developer_lump_sum_accts(
 
         # set to an empty dataframe to save memory
         orca.add_table("feasibility", pd.DataFrame())
+
+
+@orca.step()
+def subsidized_office_developer_vmt(
+    parcels, settings, scenario, coffer, buildings, year, policy,
+    add_extra_columns_func, summary):
+
+    vmt_acct_settings = policy["acct_settings"]["vmt_settings"]
+
+    if scenario in policy_setting["com_for_com_scenarios"]:
```

Comment:
@yuqiww hope I'm not missing this one too... but where is policy_setting defined?

Reply:
Ah, it should be "vmt_acct_settings" defined in line 1171. My bad!

Reply:
Just did a commit to correct it.